src/Components/Blazor/Build/test/ChildContentRazorIntegrationTest.cs

```diff
@@ -108,7 +108,7 @@ public void Render_BodyChildContent()
             Assert.Collection(
                 frames,
                 frame => AssertFrame.Component(frame, "Test.RenderChildContent", 2, 0),
-                frame => AssertFrame.Attribute(frame, RenderTreeBuilder.ChildContent, 1),
+                frame => AssertFrame.Attribute(frame, "ChildContent", 1),
```

Comment:
There were about a billion uses of this in tests. If we really care about exposing this as a constant `RenderTreeBuilder` is probably not the right place.

Reply:
It could go on `Renderer`, but I don't feel strongly that it should exist at all. It was mostly only useful for our own tests, and even then it's unclear how it's better than a string literal now we know the API is locked permanently.